config

```diff
@@ -281,6 +281,18 @@ END
             fi
             . auto/feature
         fi
+        
+        if [ $ngx_found = no ]; then
```

Comment:
Please do not include unrelated changes. Also, it's recommended to use OpenResty directly or to specify the `LUAJIT_LIB` and `LUAJIT_INC` environments properly if building the module yourself.